frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/raw-capacity-card/raw-capacity-card.tsx

```diff
@@ -51,7 +51,14 @@ const RawCapacityCard: React.FC = React.memo(() => {
   return (
     <DashboardCard>
       <DashboardCardHeader>
-        <DashboardCardTitle>{t('ceph-storage-plugin~Raw Capacity')}</DashboardCardTitle>
+        <DashboardCardTitle>
+          {t('ceph-storage-plugin~Raw Capacity')}
+          <FieldLevelHelp>
+            {t(
+              'ceph-storage-plugin~RAW capacity is the absolute total disk space available to the array subsystem.',
```

Comment:
```suggestion
              'ceph-storage-plugin~Raw capacity is the absolute total disk space available to the array subsystem.',
```

Reply:
Since this is z stream, we need to fix this in master as well.

Reply:
cc @yuvalgalanti 
PTAL.